bylaws.md

```diff
@@ -126,25 +115,27 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
 	   If no majority is achieved or if no vote achieves a majority then all standing candidates are discarded and the election for that position must be re-run.
 	   Candidates may be re-nominated.
     2. Board motions shall require a 2/3rds vote of the Board members to pass.
-7. Board meetings:
+7. Nominating:
+    1. For all nominated positions, they shall be chosen by the Board officer to whom the nominated position reports, and must be approved by a vote of 2/3rds of the Board.
```

Comment:
This sentence could be reworded to be more grammatical + easier to understand.

One candidate for replacement could be:

1. All nominated positions shall be chosen by the Board officer to whom the nominated position reports.
2. All nominated positions must be approved by a 2/3rds vote of the Board.